mer-android-chroot

```diff
@@ -88,7 +88,7 @@ bind_mount_home="yes";
 
 QUIET=yes
 
-while getopts "u:m:r:v" opt; do
+while getopts "u:m:r:vh" opt; do
```

Comment:
nitpick (optional): this could be in its small commit in case of good practice for reverts.